src/md/bylaws.md

```diff
@@ -0,0 +1,381 @@
+
+##Roles and Responsibilities
+
+Apache projects define a <a class="externalLink" href=
+"http://www.apache.org/foundation/how-it-works.html#roles">set of roles</a> with
+associated rights and responsibilities. These roles govern what tasks an individual
+may perform within the project. The roles are defined in the following sections:
+
+###Users
+
+The most important participants in the project are people who use our software.
+The majority of our developers start out as users and guide their development
+efforts from the user&rsquo;s perspective.
+
+Users contribute to the Apache projects by providing feedback to developers in
+the form of bug reports and feature suggestions. As well, users participate in the
+Apache community by helping other users on mailing lists and user support
+forums.
+
+###Contributors
+
+All of the volunteers who are contributing time, code, documentation, or
+resources to the Apex Project. A contributor that makes sustained, welcome
+contributions to the project may be invited to become a Committer, though the exact
+timing of such invitations depends on many factors.
+
+###Committers
+
+The project&rsquo;s Committers are responsible for the project&rsquo;s technical
+management. All committers have write access to the project&rsquo;s source
+repositories. Committers may cast binding votes on any technical discussion
+regarding the project.
+
+Committer access is by invitation only and must be approved by lazy consensus of
+the active PMC members. A Committer may request removal of their commit privileges
+by their own declaration. A committer will be considered
+&ldquo;emeritus/inactive&rdquo; by not contributing in any form to the project for
+over 1 year. An emeritus committer may request reinstatement of commit access from
+the PMC. Such reinstatement is subject to lazy consensus of active PMC members
+
+Commit access can be revoked by a unanimous vote of all the active PMC members
+(except the committer in question if they are also a PMC member).
+
+All Apache committers are required to have a signed Contributor License
+Agreement (<a class="externalLink" href=
+"http://www.apache.org/licenses/icla.txt">CLA</a>) on file with the Apache Software
+Foundation. There is a <a class="externalLink" href=
+"http://www.apache.org/dev/committers.html">Committer FAQ</a> which provides more
+details on the requirements for Committers
+
+A committer who makes a sustained contribution to the project may be invited to
+become a member of the PMC. The form of contribution is not limited to code. It can
+also include code review, helping out users on the mailing lists, documentation,
+etc.
+
+
+###Project Management Committee
+
+The Project Management Committee (PMC) for Apache Apex was created by a
+resolution of the board of the Apache Software Foundation on ***TBD***. The
+PMC is responsible to the board and the ASF for the management and oversight of the
+Apache Apex codebase. The responsibilities of the PMC include:
```

Comment:
I would change this to "community" instead of codebase.  The PMC is responsible for the whole community operations (like who are committers) not just for the code.